eventing/triggers/create-triggers.adoc

```diff
@@ -22,6 +22,9 @@ metadata:
   name: my-service-trigger
 spec:
   broker: default
+  filter:
+    attributes:
+      type: dev.knative.foo.bar
```

Comment:
this is OK, and we can use it

However this is the "legacy" version of filtering. the new `filters` _supersede_ the old `filter`.

However, since a) this will stay for very long time, and the `kn` cli has **NO** support for `filters` yet, it is OK 

I just wanted to mention that